xc/device.ml

```diff
@@ -1523,45 +1523,50 @@ let get_state ~xs ~qemu_domid domid =
 	with _ -> None
 
 let cmdline_of_disp info =
+	let combine_opts : bytes list option list -> bytes list = fun opts ->
```

Comment:
This is moot if I do what you asked in https://github.com/xapi-project/xenopsd/pull/272#discussion-diff-67481373